doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,267 @@
+   example, assuming the SATA disk and storage device with USB interface are
+   assigned to ``/dev/sda`` and ``/dev/sdb`` respectively, the following
+   commands set up the partition mode loadable kernel modules onto the root file
+   systems to be loaded by the pre-launched VMs:
+
+   .. code-block:: none
+
+      # Mount the Clear Linux OS root filesystem on the SATA disk
+      $ sudo mount /dev/sda3 /mnt
+      $ sudo cp -r <kernel-modules-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+      $ sudo umount /mnt
+      # Mount the Clear Linux OS root filesystem on the USB flash disk
+      $ sudo mount /dev/sdb3 /mnt
+      $ sudo cp -r <path-to-kernel-module-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+      $ sudo umount /mnt
+
+#. Copy the bootable kernel image to the /boot directory:
+
+   .. code-block:: none
+
+      $ sudo cp <path-to-kernel-image-built-in-step1>/bzImage /boot/
+
+   .. note::
+
+      If image build machine is not target board (NUC7i7DNB) which loads kernel
+      image, need to use ``scp`` copy command to update kernel image (bzImage)
+      and modules on ubuntu of NUC7i7DNB. 
+
+   .. note::
+
+      If image build machine is the target board (NUC7i7DNB) which loads kernel
+      image, need to use ``cp`` copy command to update kernel image (bzImage)
+      and modules on ubuntu of NUC7i7DNB. 
+
+Update ACRN hypervisor Image
+****************************
+
+#. Before building the ACRN hypervisor, you need to figure out the I/O
+   address of the serial port, and the PCI BDF addresses of the SATA controller
+   and the USB controllers on NUC7i7DNB board. Enter the following command to
+   get the I/O addresses of the serial port. NUC7i7DNB boards support one
+   serial port ttyS0. Need to connect the serial port to the development host,
+   in order to access the ACRN serial console to switch between pre-launched
+   VMs:
+   
+   .. code-block:: none
+
+      $ dmesg | grep ttyS0
+      [    0.000000] console [ttyS0] enabled
+      [    1.562546] 00:01: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is
+      a 16550A
+
+   The following command prints detailed information about all PCI buses and
+   devices in the system:
+
+   .. code-block:: none
+
+      $ sudo lspci -vv
+      00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21) (prog-if 30 [XHCI])
+              Subsystem: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller
+      00:17.0 SATA controller: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode] (rev 21) (prog-if 01 [AHCI 1.0])
+              Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
+      00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
+              Subsystem: Intel Corporation Ethernet Connection I219-LM
+
+   .. note:: 
+
+      Double check PCI devices BDF defined in the ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``.
```

Comment:
What should you check in this file?  How would you know if something is wrong?

Reply:
Use comment "sudo lspci -vv" when board runs unbuntu to read PCI devices BDF, these PCI devices BDF is defined on our configuration file "pci_devices.h", check it one by one.

Reply:
@dbkinder Do you have any more comment about this? Or how to make this clearer, thanks!

Reply:
I guess I'm asking what should you look at? Are you comparing the output of the lspci -vv command with information in the pci_devices.h file?  If so, then you could update this note by saying that:
```
Double check PCI devices BDF defined in the ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``
with the information reported by the ``lspci -vv`` command.
```

Reply:
Thanks you to make this clearer, we will update this according to your comments.